internal/api/api_test.go

```diff
@@ -6,6 +6,8 @@ import (
 	"net/http/httptest"
 	"testing"
 
+	"github.com/pkg/errors"
```

Comment:
Error wrapping is great, but this package may not be well-maintained. Not something to address here (we're already using it), but I'd like us to follow up with a better alternative across LDX...